Browser_IDE/editorMain.js

```diff
@@ -749,14 +749,22 @@ function audioFunctionNotification(source) {
 async function runProgram(){
     try {
         clearErrorLines();
-
-        displayEditorNotification(activeLanguageSetup.compiled ? "Compiling project..." : "Building project...", NotificationIcons.CONSTRUCTION);
-
+		
+		// the notification object returned by displayEditorNotification
+		let currentNotification = displayEditorNotification(
+			activeLanguageSetup.compiled ? "Compiling project..." : "Building project...",
+			NotificationIcons.CONSTRUCTION,-1
+		);
+		
```

Comment:
This looks good. However, the indentation uses tabs instead of spaces. If you are using VS Code, you can fix this by selecting "Convert Indentation to Spaces". I also think we should remove the empty spaces on lines 752 and 758.

<img width="735" alt="Screenshot 2024-11-26 at 1 39 58 PM" src="https://github.com/user-attachments/assets/69561f23-dae1-400d-a78e-04cee89d78e1">